### ISSUE TITLE
ValueError: 'hybrid' is not a valid EngineType

### ISSUE THREAD

**Issue by prvakt** [skodaconnect/homeassistant-myskoda#62]
I have a Superb iV car which is plug-in hybrid, and when I try to login following error will pop up. Is it possible to add this engine type to this integration?

```
Logger: custom_components.myskoda.coordinator
Source: helpers/update_coordinator.py:354
integration: MySkoda ([documentation](https://github.com/skodaconnect/homeassistant-myskoda/blob/main/README.md), [issues](https://github.com/skodaconnect/homeassistant-myskoda/issues))
First occurred: 9:08:36 AM (4 occurrences)
Last logged: 9:10:14 AM

Unexpected error fetching myskoda data
Traceback (most recent call last):
  File "<string>", line 16, in __mashumaro_from_json__
  File "/usr/local/lib/python3.12/enum.py", line 757, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/enum.py", line 1171, in __new__
    raise ve_exc
ValueError: 'hybrid' is not a valid EngineType

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 354, in _async_refresh
    self.data = await self._async_update_data()
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/myskoda/coordinator.py", line 82, in _async_update_data
    vehicle = await self.myskoda.get_vehicle(self.vin)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/myskoda/myskoda.py", line 210, in get_vehicle
    vehicle.driving_range = await self.get_driving_range(vin)
                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/myskoda/myskoda.py", line 179, in get_driving_range
    return await self.rest_api.get_driving_range(vin)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/myskoda/rest_api.py", line 77, in get_driving_range
    return await self._make_get_request(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.12/site-packages/myskoda/rest_api.py", line 41, in _make_get_request
    data = deserialize(response_text)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<string>", line 18, in __mashumaro_from_json__
mashumaro.exceptions.InvalidFieldValue: Field "car_type" of type EngineType in DrivingRange has invalid value 'hybrid'
```

**Comment by WebSpider** [comment]
Yes, we have this queued to be fixed in https://github.com/skodaconnect/myskoda/pull/70

**Comment by WebSpider** [comment]
This is released in v1.4.3, please provide your feedback!
